examples/panel-datalinks/cypress/integration/01-view-panel.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { e2e } from '@grafana/e2e';
+
+const panel = 'Datalinks';
+
+describe('viewing a panel with datalinks', () => {
+  beforeEach(() => {
+    e2e.flows.openDashboard({
+      uid: 'fTh-POZ4k',
+    });
+  });
+
+  it('should display an svg', () => {
+    e2e.components.Panels.Panel.containerByTitle(panel)
+      .should('be.visible')
```

Comment:
nit: instead of doing this we could add a test id within our panel that we can look for.